Source/Private/ARTUtils.m

```diff
@@ -0,0 +1,54 @@
+#import <time.h>
+#import <stdlib.h>
+#import "ARTUtils.h"
+#import "ARTUtils+Private.h"
+
+@implementation ARTUtils
+
+#pragma mark - Random generator
+
+static BOOL _needsSeeding = YES;
+
++ (void)seedRandomWithNumber:(unsigned int)number {
+    srandom(number);
+    _needsSeeding = NO;
+}
+
++ (void)seedRandomWithSystemTime {
+    [self seedRandomWithNumber:(unsigned)time(NULL)];
+}
+
++ (double)randomFraction {
+    long r = [self randomFromZeroToNumber:100];
```

Comment:
I'm not sure what value the intermediary local variable adds here.

```suggestion
    return [self randomFromZeroToNumber:100] / 100.0;
```

Reply:
Done.